modules/ROOT/pages/database-administration/standard-databases/create-databases.adoc

```diff
@@ -76,13 +76,16 @@ The `CREATE DATABASE` command can have a map of options, e.g. `OPTIONS {key: 'va
 | `use`
 |
 Controls how the system handles existing data on disk when creating the database.
-Currently, this is only supported with `existingDataSeedInstance` and `seedURI`, and must be set to `use`, which indicates the existing data files should be used for the new database.
+Currently, this is only supported with `existingDataSeedInstance`, `existingDataSeedServer` and `seedURI`, and must be set to `use`, which indicates the existing data files should be used for the new database.
```

Comment:
I can not make a comment above. But we should mention `existingDataSeedServer` on row 34 as well.